examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -0,0 +1,160 @@
+"""
+========================================
+Release Highlights for scikit-learn 0.23
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.23! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. For an exhaustive list of
+all the changes, please refer to the :ref:`release notes <changes_0_23>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install scikit-learn
+"""
+
+# flake8: noqa
```

Comment:
I would move that before `import numpy as np`, to avoid confusion with the install commands.

Reply:
looks like moving this at the very top does the trick